ark/internal/genai/team.go

```diff
@@ -206,6 +213,12 @@ func (t *Team) executeWithTracking(tracker *OperationTracker, execFunc func(cont
 
 // executeMemberAndAccumulate executes a member and accumulates new messages
 func (t *Team) executeMemberAndAccumulate(ctx context.Context, member TeamMember, userInput Message, messages, newMessages *[]Message, turn int) error {
+	// Add team and current member to execution metadata for streaming
+	ctx = WithExecutionMetadata(ctx, map[string]interface{}{
+		"team":  t.Name,
```

Comment:
we could avoid the hard coding and use teamKey or similar right? across all context usage?

Reply:
keys could be in the context file